junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala

```diff
@@ -108,7 +108,7 @@ class ScalaJSJUnitPlugin(val global: Global) extends NscPlugin {
     import global._
 
     val phaseName: String = "junit-inject"
-    val runsAfter: List[String] = List("mixin")
+    val runsAfter: List[String] = List("mixin", "junit-make-mixin-bridges")
```

Comment:
This change is not necessary, since the mixin plugin takes care of that. And we've discovered that `runsAfter` warns for missing dependencies (and errors out with fatal-warnings).